src/Form/Form.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react'
+import { Route, Switch } from 'react-router-dom'
+import { Link, NavLink } from 'react-router-dom';
+import './Form.scss';
+import ExchangeContainer from '../ExchangeContainer/ExchangeContainer.js'
+
+
+const Form = (props) => {
+  const [userCurrency, setUserCurrency] = useState('')
+  const [userAmount, setUserAmount] = useState('')
+  const [newCurrency, setNewCurrency] = useState('')
+
+  const submitExchangeData = event => {
+    event.preventDefault()
+    const newExchange = {
+      id: Date.now(),
+      userCurrency, 
+      userAmount,
+      newCurrency
```

Comment:
I didn't realize I'd done so much of Form ... Sarah, remember to do atomic commits! :)